accessing
recordedException: anObject

	^ actualTrace recordedException: anObject